lib/api/core/validation/index.js

```diff
@@ -756,7 +774,7 @@ function getParent(structuredFields, fieldPath) {
 
   for (i = 0; i < fieldPath.length - 1; i++) {
     if (!pointer.children.hasOwnProperty(fieldPath[i])) {
-      throw new InternalError(`The parent field of the field "${fieldPath.join('.')}" is not defined.`);
+      throw new KuzzleInternalError(`The parent field of the field "${fieldPath.join('.')}" is not defined.`);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 110 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)